aws/resource_aws_cloudtrail.go

```diff
@@ -227,6 +275,16 @@ func resourceAwsCloudTrailRead(d *schema.ResourceData, meta interface{}) error {
 	}
 	d.Set("enable_logging", logstatus)
 
+	// Get EventSelectors
+	eventSelectorsOut, err := conn.GetEventSelectors(&cloudtrail.GetEventSelectorsInput{
+		TrailName: aws.String(d.Id()),
+	})
+	if err != nil {
+		return err
+	}
+
+	d.Set("event_selector", flattenAwsCloudTrailEventSelector(eventSelectorsOut.EventSelectors))
```

Comment:
We should check and return for errors here since it is not a simple `string` attribute:

```
if err := d.Set("event_selector", flattenAwsCloudTrailEventSelector(eventSelectorsOut.EventSelectors)); err != nil {
  return err
}
```